src/gui_common/MainMenu.cs

```diff
@@ -29,9 +29,11 @@ public class MainMenu : Node
     [Export]
     public NodePath FreebuildButtonPath;
 
+    [Export]
+    public NodePath GLES2PopupPath;
```

Comment:
This doesn't run for me. I think you forgot to include the change to include this in the main menu scene

Reply:
nodepath got messed up, will add back